### DISCUSSION TITLE
A DNS record has been found but its syntax is not valid: Public key mismatchs

### DISCUSSION THREAD

**Discussion by arobase13** [modoboa/modoboa#2182] [Help]
Hello,
new to Modoboa, I am having a problem for DKIM.
I still get this message
A DNS record has been found but its syntax is not valid: Public key mismatchs
I did create a dsn record like example._domainkey.domain.com but it doesn't work.
The public key generated by modoboa, I tested it on https://dkimcore.org/c/keycheck and it is invalid.
This is not a good DKIM key record. You should fix the errors shown in red
How to generate a new key?
thank you in advance

**Comment by chocolaid** [comment]
Hi, did you fix this iissue?
